tests/catalyst/callbacks/test_checkpoint.py

```diff
@@ -152,52 +159,39 @@ def handle_batch(self, batch):
         self.batch = {"features": x, "targets": y, "logits": logits}
 
 
-def test_device_load_on_stage_start():
-    to_check_devices = ["cpu"]
-    for device in to_check_devices:
-        with TemporaryDirectory() as logdir:
-            runner = CustomRunner(logdir, DeviceEngine(device))
-            runner.run()
+def to_check_devices():
+    cuda_devices = [f"cuda:{i}" for i in range(NUM_CUDA_DEVICES)]
+    return ["cpu", *cuda_devices]
```

Comment:
🤔  should we simplify to `cpu` and `cuda` only? if it works on one GPU, we suppose, that it will work on the another too ;)

Reply:
i reformatted old test, but sure, we can simplify it